package/com.unity.formats.usd/Editor/Scripts/UsdBuildPostProcess.cs

```diff
@@ -39,6 +39,10 @@ public static void OnPostprocessBuild(BuildTarget target, string pathToBuiltProj
             {
                 destination = pathToBuiltProject.Replace(".exe", "_Data/Plugins");
             }
+            else
```

Comment:
I would throw an exception.
You don't want silent errors on crazy platforms.

Reply:
Is it really an issue to build a project on mobile platform with the usd package, even if not used?

Reply:
ppl should know that they USD was not used for the iOS build.
I think I agree with you that stopping the build might be intense, but the very least there should be message about the fact that device builds are not supported.

Reply:
Good point, I'll add an extra message

Reply:
I added a warning message on non desktop builds.